dockstore-webservice/src/main/java/io/dockstore/webservice/core/webhook/InstallationRepositoriesPayload.java

```diff
@@ -42,21 +43,48 @@ public class InstallationRepositoriesPayload extends Payload {
                       "full_name": "dockstore/dockstore-ui2"
                     }
                   ]
+                  "repositories_removed": [
+                  ]
                 }
                 """;
 
+    @JsonProperty
+    @Schema(name = "action", description = "The action which the event describes", requiredMode = RequiredMode.REQUIRED)
+    private String action;
+
     @JsonProperty("repositories_added")
     @Schema(name = "repositories_added", description = "An array of repository objects, which were added to the installation", requiredMode = RequiredMode.REQUIRED)
     private List<WebhookRepository> repositoriesAdded;
 
+    @JsonProperty("repositories_removed")
+    @Schema(name = "repositories_removed", description = "An array of repository objects, which were removed from the installation", requiredMode = RequiredMode.REQUIRED)
```

Comment:
TIL they're working on an openapi 4 for things like this (i.e. either this parameter or this other one https://github.com/OAI/OpenAPI-Specification/issues/256#issuecomment-1908868607 )

i.e. it's a little weird that both `repositoriesRemoved` and `repositoriesAdded` are required